test/multiverse/suites/sinatra/sinatra_classic_test.rb

```diff
@@ -5,7 +5,6 @@
 require 'sinatra'
 require_relative 'sinatra_test_cases'
 require_relative '../../../helpers/exceptions'
-SimpleCov.command_name "test:multiverse[sinatra]"
```

Comment:
I think we accidentally deleted this one

Reply:
For this and the other deleted ones, I moved them to a different file: the first appearing 'test' file so that we have consistency across where `command_name` is found in the suite :)

Reply:
Awesome! Thanks for clarifying!